subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/dependencies/FindAndroidAppTask.kt

```diff
@@ -70,16 +72,17 @@ public abstract class FindAndroidAppTask @Inject constructor(
     }
 
     public object Options {
+
         @Suppress("EnumEntryName")
         public enum class CONFIGURATION {
             main {
-                override fun mapToType(): Main = Main
+                override fun mapToTypes(): Set<ConfigurationType> = setOf(Main)
             },
             android_test {
-                override fun mapToType(): AndroidTests = AndroidTests
+                override fun mapToTypes(): Set<ConfigurationType> = setOf(AndroidTests, Main)
```

Comment:
Temporary workaround for implicit dependency between configurations.
Will detect it implicitly by configuration hierarchy.